### PR TITLE
CDCI-27 Email Notification Implementation

### PR DESCRIPTION
**Pull request by venkatasaipunnam**

### CODE REVIEW

File: src/main/java/com/clarku/workshop/utils/EmailConstants.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.clarku.workshop.utils;
+
+public class EmailConstants {
+
+	public static final String WELCOME_SUBJECT = "Welcome to Workshop Application";
+	public static final String SIGNUP_SUCCESS = "signUpSuccess";
```
Comment (ven-sai) [inline src/main/java/com/clarku/workshop/utils/EmailConstants.java]: Update to SIGNUP_SUCCESS_TEMPLATE

File: src/main/java/com/clarku/workshop/utils/EmailHelper.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+	@Autowired
+	JavaMailSenderImpl mailer;
+	
+	@Autowired
+	TemplateEngine templateEngine;
+	
+	public void sendEMail(EmailVO emailVO) throws GlobalException, EmailException {
+		MimeMessage mimeMessage = mailer.createMimeMessage();
+        MimeMessageHelper helper = new MimeMessageHelper(mimeMessage, "UTF-8");
```
Comment (ven-sai) [inline src/main/java/com/clarku/workshop/utils/EmailHelper.java]: Check the format

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1 +1,15 @@
+server.error.include-stacktrace=never
+server.port=8090
 
+#Update accordingly
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.username=postgres
+spring.datasource.password=9290462177
```
Comment (ven-sai) [inline src/main/resources/application.properties]: remove the passwords

File: src/main/java/com/clarku/workshop/utils/EmailHelper.java (hunk 2 above)
Comment (venkatasaipunnam) [reply src/main/java/com/clarku/workshop/utils/EmailHelper.java]: Updated the format

File: src/main/resources/application.properties (hunk 3 above)
Comment (venkatasaipunnam) [reply src/main/resources/application.properties]: Updated

**Review by ven-sai — APPROVED** [review]

**Pull request merged**